pkg/volume/csi/csi_mounter.go

```diff
@@ -107,49 +111,97 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 		return nil
 	}
 
-	csiSource, err := getCSISourceFromSpec(c.spec)
-	if err != nil {
-		klog.Error(log("mounter.SetupAt failed to get CSI persistent source: %v", err))
-		return err
-	}
-
 	csi := c.csiClient
 	ctx, cancel := context.WithTimeout(context.Background(), csiTimeout)
 	defer cancel()
 
-	// Check for STAGE_UNSTAGE_VOLUME set and populate deviceMountPath if so
-	deviceMountPath := ""
-	stageUnstageSet, err := csi.NodeSupportsStageUnstage(ctx)
+	volSrc, pvSrc, err := getSourceFromSpec(c.spec)
 	if err != nil {
-		klog.Error(log("mounter.SetUpAt failed to check for STAGE_UNSTAGE_VOLUME capabilty: %v", err))
+		klog.Error(log("mounter.SetupAt failed to get CSI persistent source: %v", err))
 		return err
 	}
 
-	if stageUnstageSet {
-		deviceMountPath, err = makeDeviceMountPath(c.plugin, c.spec)
-		if err != nil {
-			klog.Error(log("mounter.SetUpAt failed to make device mount path: %v", err))
-			return err
+	driverName := c.driverName
+	volumeHandle := c.volumeID
```

Comment:
nit: `volumeHandle` == `volumeID`? Maybe we should rename the `volumeID` to `volumeHandle` on the `csiMountMgr`

Reply:
yeah it should be standard everywhere.